ner/scripts/benchmarks.sh

```diff
@@ -42,7 +42,7 @@ for JAR in `ls $LIB/*.jar`; do
     cpath="$cpath:$JAR"
 done
 
-CMD="java -classpath  ${cpath} -Xmx12g edu.illinois.cs.cogcomp.ner.NerBenchmark $1 $2 $3 $4 $5"
+CMD="java -classpath  ${cpath} -Xmx14g -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=8001 -agentpath:/Applications/YourKit-Java-Profiler-2017.02.app/Contents/Resources/bin/mac/libyjpagent.jnilib edu.illinois.cs.cogcomp.ner.NerBenchmark $1 $2 $3 $4 $5"
```

Comment:
"Xmx12g" -> "Xmx14g"? I thought this change will reduce the memory footprint.

Reply:
@danyaljj That actually depends on several things: is the model they load pruned, is the model one of ours or their own custom model, how many features remain, what gaz do they use, how many examples do they preload. I increased this because I was having failures when loading the unpruned ontonotes. We could reduce this, but it is the max, not the min, so I think it's OK.